internal/service/projectapikey/resource_project_api_key.go

```diff
@@ -132,12 +125,7 @@ func resourceMongoDBAtlasProjectAPIKeyCreate(ctx context.Context, d *schema.Reso
 		return diag.FromErr(fmt.Errorf("error setting `private_key`: %s", err))
 	}
 
-	firstProjectID, err := getFirstProjectIDFromAssignments(d)
-	if err != nil {
-		return diag.FromErr(fmt.Errorf("could not obtain a project id from state: %s", err))
-	}
 	d.SetId(conversion.EncodeStateID(map[string]string{
-		"project_id": *firstProjectID, // defined to avoid breaking changes (preserve id format), can be removed when root project_id is removed.
```

Comment:
Should we mention this id format change in a new changelog entry? Just for transparency

Reply:
or the guide?

Reply:
don't know, it's invisible to the user, and migration test worked fine

Reply:
true, don't see it would impact the user as our implementation is not relying on the id to obtain any information.